plugins/postcss-nesting/.tape.mjs

```diff
@@ -6,7 +6,7 @@ const mixinPluginRule = () => {
 		postcssPlugin: 'mixin',
 		AtRule: {
 			mixin(node, { postcss }) {
-				node.replaceWith(postcss().process('& .in{ &.deep { color: blue; }}', {from : 'mixin.css'}).root);
+				node.replaceWith(postcss.parse('& .in{ &.deep { color: blue; }}', {from : 'mixin.css'}));
```

Comment:
This was triggering PostCSS to warn about now having any plugins.
Using `.parse` resolves this.